react/components/Notification/NotificationInline.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { memo } from 'react'
+
+import NotificationContent from './NotificationContent'
+
+interface Props {
+  content?: string
+}
+
+function NotificationInline({ content = '' }: Props) {
+  if (!content) {
+    return null
+  }
+
+  return <NotificationContent content={content} />
+}
+
+NotificationInline.schema = {
+  title: 'admin/editor.notification-bar.title',
+}
+
+export default memo(NotificationInline)
```

Comment:
Static properties are lost here.

You can do something like:

```tsx
const MemoNoticationInline = memo(NotificationInline)

MemoNotificationInline.schema = {
  title: 'admin/editor.notification-bar.title',
}

export default MemoNotificationInline
```

Reply:
Nice! Did not know that!